controller/linodemachine_controller_helpers.go

```diff
@@ -50,6 +50,12 @@ import (
 // The decoded user_data must not exceed 16384 bytes per the Linode API
 const maxBootstrapDataBytes = 16384
 
+var (
+	errNoPublicIPv4Addrs      = errors.New("no public ipv4 addresses set")
+	errNoPublicIPv6Addrs      = errors.New("No public IPv6 address set")
+	errNoPublicIPv6SLAACAddrs = errors.New("No public SLAAC address set")
```

Comment:
Don't capitalize error strings. There is a linter that should be checking this.

```suggestion
	errNoPublicIPv6Addrs      = errors.New("no public IPv6 address set")
	errNoPublicIPv6SLAACAddrs = errors.New("no public SLAAC address set")
```

Reply:
done